tests/localnet/conftest.py

```diff
@@ -13,3 +15,57 @@ def algod_health_fast_timings(mocker: MockerFixture) -> None:  # noqa: ignore[PT
 @pytest.fixture()
 def health_success(httpx_mock: HTTPXMock) -> None:  # noqa: ignore[PT004]
     httpx_mock.add_response(url=ALGOD_HEALTH_URL)
+
+
+@pytest.fixture()
+def _localnet_up_to_date(proc_mock: ProcMock, httpx_mock: HTTPXMock) -> None:
+    proc_mock.set_output(
+        ["docker", "image", "inspect", ALGORAND_IMAGE, "--format", "{{.RepoDigests}}"],
+        ["[algorand/algod@sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa]"],
+    )
+
+    proc_mock.set_output(
+        ["docker", "image", "inspect", INDEXER_IMAGE, "--format", "{{.RepoDigests}}"],
+        ["[makerxau/algorand-indexer-dev@sha256:bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb]"],
+    )
+
+    httpx_mock.add_response(
+        url="https://registry.hub.docker.com/v2/repositories/makerxau/algorand-indexer-dev/tags/latest",
+        json={
+            "digest": "sha256:bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb",
+        },
+    )
+
+    httpx_mock.add_response(
+        url="https://registry.hub.docker.com/v2/repositories/algorand/algod/tags/latest",
+        json={
+            "digest": "sha256:aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+        },
+    )
+
+
+@pytest.fixture()
+def _localnet_out_of_date(proc_mock: ProcMock, httpx_mock: HTTPXMock) -> None:
```

Comment:
In contrast with `localnet_up_to_date`, `_localnet_out_of_date` is only used by a single test case in `test_localnet_start.py`. Is there any need to have it defined in conftest in this case?